onegov/election_day/formats/election/wabstic_majorz.py

```diff
@@ -79,6 +81,16 @@ def import_election_wabstic_majorz(
     entities = principal.entities[election.date.year]
     election_id = election.id
 
+    def has_no_lines(lines, filename):
+        if not list(lines):
```

Comment:
You consume the whole list of lines, to check if there's at least one line. That seems wasteful. If lines is not a generator, you can just check for by running `if lines`.

If lines is a generator, don't have to consume it to check for it. You can do either this:
```python
for line in lines:
     return False  # has lines

return True # has no lines
```
Or by getting the next value of the generator with a fallback:
```python
if next(lines, None):
    return True # has lines

return False  # has no lines

Reply:
Yes, it's definitively a blocker...